### PR TITLE
 DM-38164: Update documentation to match change to dark defaults.

### PR DESCRIPTION
**Pull request by czwa**
Use multiple exposure times for daily dark default settings.

### CODE REVIEW

File: python/lsst/ts/externalscripts/base_make_calibrations.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ def get_schema(cls):
                       minimum: 0
                   - type: number
                     minimum: 0
-                default: 5
+                default: [5,  5, 5, 5, 5, 15, 15, 30, 30, 30, 30,  30, 30, 30, 30,  30, 30, 30, 30,  30, 30, 30, 30]
```
Comment (patrickingraham) [inline python/lsst/ts/externalscripts/base_make_calibrations.py:229]: This is throwing a flake8 error in the CI testing.

**Review by patrickingraham — COMMENTED** [review]
Basically this is fine, minus the flake8 bit.
I'd like to have an operator run the config before merging though just to make sure the script will handle it properly.

**Comment by tribeiro** [comment]
Please, remember to update the version history!

File: python/lsst/ts/externalscripts/base_make_calibrations.py (hunk 1 above)
Comment (czwa) [reply python/lsst/ts/externalscripts/base_make_calibrations.py]: Fixed, and version history updated.  I'm 100% on board with having a test pass before merging.

**Comment by patrickingraham** [comment]
Tests passed (twice). Please rebase (if necessary) and squash the child commits into a single commit message.
Thanks!

**Review by patrickingraham — APPROVED** [review]
Sorry, I thought I approved this already.

**Pull request merged**